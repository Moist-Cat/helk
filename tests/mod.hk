let a = 42 in
    if (a % 2) {
        prints("Odd");
    }
    else {
        print(a);
        prints("Even");
    };
