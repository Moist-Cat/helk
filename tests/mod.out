42.000000
Even
